### PR TITLE
New start screen

### PR DESCRIPTION
**Pull request by taleldayekh**

### CODE REVIEW

File: raiden-dapp/src/components/AppCore.vue [hunk 1]
```diff
@@ -1,13 +1,25 @@
 <template>
-  <tokens />
+  <NoTokens v-if="!tokens.length" />
```
Comment (kelsos) [inline raiden-dapp/src/components/AppCore.vue]:
```suggestion
  <no-tokens v-if="!tokens.length" />
```

In all the components we use the components in kebab case.
We should also add a lint rule to enforce this.

File: raiden-dapp/src/components/NoTokens.vue [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+import { Component, Mixins } from 'vue-property-decorator';
+import NavigationMixin from '@/mixins/navigation-mixin';
+
+@Component({})
+export default class NoTokens extends Mixins(NavigationMixin) {}
+</script>
+
+<style lang="scss" scoped>
+.new-token {
+  text-align: center;
```
Comment (kelsos) [inline raiden-dapp/src/components/NoTokens.vue]: Can you move the button and header parts here?
as `&__button ` and `&__header`?

File: raiden-dapp/src/components/AppCore.vue (hunk 1 above)
Comment (taleldayekh) [reply raiden-dapp/src/components/AppCore.vue]: Done

File: raiden-dapp/src/components/NoTokens.vue (hunk 2 above)
Comment (taleldayekh) [reply raiden-dapp/src/components/NoTokens.vue]: Done

**Pull request merged**